computation/src/main/java/com/powsybl/computation/ComputationParameters.java

```diff
@@ -30,24 +33,10 @@
      */
     OptionalLong getDeadline(String commandId);
 
-    Optional<String> getQos(String commandId);
```

Comment:
Does it break something?

Reply:
> Does it break something?

Currently, no one else uses it, except me.